releasenotes.md

```diff
@@ -0,0 +1,9 @@
+# design.automation.inventor-utilities
+
+# 1.0.0-beta3
+
+* modified requestOrContentFile onDemand parameter to be optional for now, usage of this parameter (other value than empty ) will produce exception
```

Comment:
Lets change the value of version to be 1.1.0-beta. Just because we changed an interface.

Reply:
We should also make more visible, that we changed contract which will break the usage.